ThirdPartyModules/Mirasvit/Rewards.php

```diff
@@ -268,19 +268,52 @@ private function getMirasvitRewardsAmount($quote)
                 $websiteId = $quote->getStore()->getWebsiteId();
                 $rules = $this->mirasvitRewardsSpendRulesListHelper->getRuleCollection($websiteId, $customer->getGroupId());
                 if ($rules->count()) {
+                    $totalAmount = 0;
+                    $totalPoints = 0;
+        
+                    $mockSubtotal = 999999999;
```

Comment:
Can you explain why set the value to 999999999 here?

Reply:
@johnmccombs1 Cause we always try to apply full available rewards amount to Bolt cart, we create a mock subtotal large enough to implement this purpose.

Reply:
@dumega Can we create a class constant instead of a variable for this number?

Reply:
@vitaliyreznikov Done